litmus-portal/frontend/src/views/Analytics/Overview/index.tsx

```diff
@@ -227,12 +221,13 @@ const Overview: React.FC = () => {
           link={
             <Link
               underline="none"
-              color="primary"
               href="https://prometheus.io/docs/introduction/overview/"
```

Comment:
Please update the Prometheus link, we do not have to point to official docs @vanshBhatia-A4k9

Reply:
can you provide the appropriate link?

Reply:
https://github.com/litmuschaos/litmus/tree/master/monitoring#model-1-optional-prometheus-scrape-config-model

Reply:
Use the above link @vanshBhatia-A4k9